engines/bastion_katello/app/assets/javascripts/bastion_katello/files/views/files.html

```diff
@@ -0,0 +1,39 @@
+<span page-title>{{ 'Files' | translate }}</span>
+
+<div data-extend-template="layouts/table-with-header.html">
+  <div data-block="header" translate>
+    Files
+  </div>
+
+  <span data-block="no-rows-message" translate>
+    You currently don't have any Files.
+  </span>
+
+  <span data-block="no-search-results-message" translate>
+    Your search returned zero Files.
+  </span>
+
+  <div data-block="table">
+    <table class="table table-striped table-bordered" ng-class="{'table-mask': table.working}">
+      <thead>
+        <tr bst-table-head>
+          <th bst-table-column="name">{{ "Name" | translate }}</th>
```

Comment:
This is the form we use everywhere else for table headers, IIRC due to how the directives clash on transclusion.